pkg/cmd/server/origin/ensure.go

```diff
@@ -93,57 +98,57 @@ func (c *MasterConfig) ensureComponentAuthorizationRules() {
 	}
 }
 
-// ensureDefaultNamespaceServiceAccountRoles initializes roles for service accounts in the default namespace
-func (c *MasterConfig) ensureDefaultNamespaceServiceAccountRoles() {
+// ensureNamespaceServiceAccountRoleBindings initializes roles for service accounts in the namespace
+func (c *MasterConfig) ensureNamespaceServiceAccountRoleBindings(namespaceName string) {
 	const ServiceAccountRolesInitializedAnnotation = "openshift.io/sa.initialized-roles"
 
 	// Wait for the default namespace
```

Comment:
I wonder if we should wait for the default namespace before calling this function